packages/esm-patient-attachments-app/package.json

```diff
@@ -38,7 +38,7 @@
   },
   "dependencies": {
     "@carbon/react": "^1.76.0",
-    "@openmrs/esm-patient-common-lib": "^8.2.0",
+    "@openmrs/esm-patient-common-lib": "next",
```

Comment:
@ibacher is this accurate?

Reply:
Normally, I think adding these things to depedencies (when they are in peerDependencies) is incorrect, but the version number `next` is more correct than `^8.2.0`, since we're on 9.x now.